rest/index.py

```diff
@@ -521,6 +545,44 @@ def kibanaDashboards():
     return json.dumps(resultsList)
 
 
+def getWatchlist():
+    WatchList = []
```

Comment:
There's some inconsistencies for this variable name. You define it here with a capital 'W', but then later on, you reference it using the lowercase version. I think we should use the lowercase variable name throughout.

Reply:
these are two different variables. index.py uses python based WatchList variable for it's database results. The other is node based variables, which do slightly different things.

Reply:
I'm specifically referring to https://github.com/mozilla/MozDef/pull/906/files#diff-c9dff09105e9694576c5a91b26bcd704R549 and https://github.com/mozilla/MozDef/pull/906/files#diff-c9dff09105e9694576c5a91b26bcd704R554 (both of which are in index.py).

Reply:
I've modified this in a way that these two vars no longer imitate one another